experimental/PBWDeformations/GapWrapper.jl

```diff
@@ -0,0 +1,67 @@
+function liealgebra_struct_consts_gap(R::Ring, dynkin::Tuple{Char,Int})
+  @req is_valid_dynkin(dynkin...) "Input not allowed by GAP."
+  @req R == QQ "Works currently only for QQ." # TODO
+
+  GAPG = GAP.Globals
+
+  L = GAPG.SimpleLieAlgebra(GAP.julia_to_gap(string(dynkin[1])), dynkin[2], GAPG.Rationals)
+  dimL = GAPG.Dimension(L)
+  comm_table_L = GAP.gap_to_julia(GAPG.StructureConstantsTable(GAPG.Basis(L)))[1:dimL]
```

Comment:
Avoid `GAP.gap_to_julia`, better to specify an explicit target Julia type. E.g. 
```suggestion
  comm_table_L = Vector{Any}(GAPG.StructureConstantsTable(GAPG.Basis(L)))[1:dimL]
```
or even like this (note that I moved `[1:dimL]`)
```suggestion
  comm_table_L = Matrix{NTuple{2,Vector{Int}}}(GAPG.StructureConstantsTable(GAPG.Basis(L))[1:dimL])
```
That way the code is type stable, and also self-documenting ("what kind of object is `comm_table_L`?" can now easily be answered)